.github/workflows/package.yml

```
@@ -25,24 +25,22 @@ jobs:
     strategy:
       fail-fast: false
     steps:
-    
-    - name: Checkout code 
-      uses: actions/checkout@v2
+    - name: Checkout code
+      uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)